.travis.yml

```diff
@@ -9,6 +9,10 @@ php:
   - 7.4
   - nightly
 
+matrix:
+  allow_failures:
+    - php: nightly
```

Comment:
`nightly` is out of date and 8.0snapshot is not published yet  - see https://github.com/travis-ci/php-src-builder/pulls